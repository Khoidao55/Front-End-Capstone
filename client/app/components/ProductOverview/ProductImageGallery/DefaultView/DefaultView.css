.default-image {
  max-width: 700px;
  max-height: 700px;
  /* align-items: center; */
  /* width: auto;
  display: flex;
  height: auto;
  justify-content: center;
  margin-left:10%;
  margin-top: 10%; */
  z-index: 2;
}


/* .display-image {
  display: flex;
  cursor: crosshair;
  max-width:100%;
  max-height:100%;
  height: auto;
  width:auto;
  z-index: 2;
} */

.display-image{
  display: block;
  margin-left: auto;
  margin-right: auto;
  cursor: crosshair;
  max-width: 100%;
  max-height: 100%;
  /* margin: 0px; */
  /* position: relative;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%); */
  z-index: 2;
}

