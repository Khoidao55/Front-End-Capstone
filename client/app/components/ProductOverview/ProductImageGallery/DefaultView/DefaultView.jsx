import React from 'react';
import ReactImageMagnify from 'react-image-magnify';
import './DefaultView.css';

const DefaultView = ({ onClickImage, defaultPhoto }) => {
  if (defaultPhoto !== undefined) {
    return (
      <div className="default-image">
        {/* <div onClick={onClickImage} className="image">
          <ReactImageMagnify {...{
            smallImage: {
              isFluidWidth: true,
              src: defaultPhoto.url,
            },
            largeImage: {
              src: defaultPhoto.url,
              width: 1200,
              height: 1800,
            },
          }}
          />
        </div> */}
          <img className="display-image" onClick={onClickImage} alt="My Main Product Display" src={defaultPhoto.url} />
      </div>
    );
  }
  return (
    <div />
  );
};

export default DefaultView;
