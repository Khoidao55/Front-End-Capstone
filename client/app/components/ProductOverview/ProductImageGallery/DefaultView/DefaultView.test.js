import React from 'react';
import { shallow } from 'enzyme';
import DefaultView from './DefaultView.jsx';

const dummyProps = {
  onClickImage: () => { console.log('click the image'); },
  defaultPhoto: {},
};

describe('Product Image Default View', () => {
  test('should render my component', () => {
    const wrapper = shallow(<DefaultView {...dummyProps} />);

    expect(wrapper).toMatchSnapshot();
  });
});
