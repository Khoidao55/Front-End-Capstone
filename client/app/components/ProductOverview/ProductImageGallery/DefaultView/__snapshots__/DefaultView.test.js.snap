// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default View should render my component 1`] = `ShallowWrapper {}`;

exports[`Product Image Default View should render my component 1`] = `ShallowWrapper {}`;
